### PR TITLE
APM Onboarding Nav

### PR DESCRIPTION
**Pull request by brett0000FF**
### What does this PR do? What is the motivation?

Restructure Sending Traces to Datadog navigation. [Staging Link](https://docs-staging.datadoghq.com/brett0000FF/DOCS-6420-nav/tracing/trace_collection/).

- Rename Sending Traces to Datadog.
   **Note**: We could rename this to _Getting Started_ potentially.
- Remove Local Library Injection.
- Remove Instrumenting with Datadog Tracing Libraries.
  **Note**: We could leave it in the nav and title it _Manual Instrumentation_ potentially.
- Consolidate Custom Instrumentation pages in nav.
- Create new parent page for general Custom Instrumentation.

| Before | After |
|--------|--------|
| image | image

| 

TODO: 
- [x] Fix broken links.

### Merge instructions

- [ ] Please merge after reviewing

### CODE REVIEW

File: content/en/tracing/trace_collection/custom_instrumentation/_index.md [hunk 1]
```diff
@@ -29,23 +14,38 @@ algolia:
...
 
-{{< partial name="apm/apm-manual-instrumentation.html" >}}
+## Getting started
 
+Follow the relevant documentation for your custom instrumentation approach to learn more:
 
-<br>
+- [Datadog libraries][2]
+- [OpenTelemetry][3]
+- [OpenTracing][4]
```
Comment (kayayarai) [inline content/en/tracing/trace_collection/custom_instrumentation/_index.md:29]: Let's put these in the same order as in the nav:
- OTel
- Datadog
- OpenTracing - and maybe add a "this project is not supported anymore but might work for you if no other method works" (or maybe remove it from this list?)

**Review by kayayarai — CHANGES_REQUESTED** [review]
Just one minor suggestion for reordering, and the following feedback:
- the language tiles in content/en/tracing/trace_collection/_index.md are pointing at dd-libraries instructions -- maybe we should point them either at OTel, or remove the tiles and present the options for instrumenting again (OTel, dd-libraries, OpenTracing, etc)
- content/en/tracing/_index.md could use some updating with the changes you've made: getting rid of "Sending traces" and changing "Connect" to "Correlate"

File: content/en/tracing/trace_collection/custom_instrumentation/_index.md (hunk 1 above)
Comment (brett0000FF) [reply content/en/tracing/trace_collection/custom_instrumentation/_index.md]: Great feedback--thanks, @kayayarai ! This should all be fixed, in addition to some suggestions from Irene and Vidur.

**Review by kayayarai — APPROVED** [review]
💥

**Pull request merged**